queries.py

```diff
@@ -5,6 +5,7 @@
 from helpers import log
 from datetime import datetime
 from pytz import timezone
+from .sudo_query_helpers import query
```

Comment:
Not sure where I can import this, unless the function using it is separated in an other file so we import the logic here instead.

Reply:
try to fit the original pattern (even if you don't agree) queries here are meant some kind of construct thing, not an execution thing